src/libs/state.js

```diff
@@ -684,7 +684,9 @@ const state = new Vue({
             }
 
             if (isActiveBuffer) {
-                this.setActiveBuffer(network.id, network.serverBuffer().name);
+                let target = (network.buffers.length > 1) ?
+                    network.buffers[1].name : network.serverBuffer().name;
```

Comment:
This ternary should use the existing format used throughout the code:
~~~
let target = expr ?
    val1 :
    val2;
~~~